frontend/src/main/java/puf/frisbee/frontend/core/Constants.java

```diff
@@ -5,4 +5,5 @@
     public static final int SCENE_HEIGHT = 720;
     public static final double CHARACTER_HEIGHT = 200;
     public static final double CHARACTER_WIDTH = 172;
+    public static final int TEAM_LIVES = 5;
```

Comment:
Ich bin mir nicht ganz sicher, ob das wirklich eine Konstante sein sollte 🤔 gehört das nicht eher zum Game Model?

Reply:
Ja, ich bin mir da auch unsicher. Kann es gerne ändern.

Reply:
Aber da fällt mir auf: Warum sind denn die im Model definierten Variablen nicht 'static final'? Die werden doch auch nicht geändert. Sind daher ja doch auch Konstanten, oder?

Reply:
Und werden denn die Leben nicht bei jedem neuen Level wieder zurückgestetzt auf fünf Leben? Dann müsste dies doch eher beim TeamModel bleiben. Das Game bezieht sich ja auf mehrere Level.

Reply:
Die Anzahl der Leben wird ja mit jedem Level zurückgesetzt - das Team wird ja auch nicht neu instanziiert bei jedem Level. Also dann gehört es doch ins Level? (gerne auch final, wenn wir bei jedem Level die gleiche Anzahl an Leben laden).
Bei den anderen Settings im Model war ich mir auch nicht sicher, ob wir die in der DB speichern wollen, können die aber gerne auch final machen.

Reply:
Es macht schon mehr Sinn, die (verbliebenen) Leben mit dem Level zu speichern, und die geschafften Level auch mit dem Game. So könnte man das Spiel wohl auch zu späterer Zeit am letzten Spielstand wieder aufnehmen. Dann sind aber die Variablen 'teamLevel' und 'teamScore' auch irgendwie fraglich … oder?

Reply:
Andererseits werden für das Team ja wohl auch nur die Leben des letzten geschafften Levels benötigt. Wenn also nur ein Wert gespeichert werden muss, könnte man das schon auch im TeamModel machen.

Reply:
Die verbliebenen Leben müssen doch nicht gespeichert werden - ein Spiel kann ja nur immer wieder mit einem "frischen" Level geladen werden, also immer mit neuen Leben. Das muss also nicht persistent gespeichert werden.
`teamLevel` muss bei jedem Team, bzw dem Teamspielstand gespeichert werden, damit wir auch später wieder laden können - ebenso wie der Score. Das macht meiner Meinung nach schon Sinn beim Team (in der Datenbank wäre es aber ein extra table). Oder bin ich jetzt verwirrt?

Reply:
OK. Ich dachte, wir holen alles aus der Datenbank: Lives, Level und Score.

Reply:
Level und Score ja, Lives müssen wir ja nicht speichern, wenn wir immer mit 5 bei jedem Spiel anfangen, oder?

Reply:
Und ganz am Anfang dachte ich auch, wir speichern das alles in einer GameTabelle in der jedes Level mit seinen Attributen gelistet ist.

Reply:
Achso, du meinst, dass jedes Level mit einer anderen Anzahl an Leben anfangen kann? Das ist auch eine Idee... aber egal ob DB oder nicht (erstmal egal zumindest) - es gehört jetzt erstmal zum Level Model, oder?

Reply:
Nö, nicht dass man jedes Level mit einer anderen Anzahl an Leben anfangen kann. Aber dass man halt die Information hat, bei welchem Level man mit welchem Spielstand wieder einsteigen kann. So dachte ich.

Reply:
Aber das könnte man auch lassen, man spielt halt nur auf dem 'unfertigen' Level weiter.